src/main/java/qna/domain/Answer.java

```diff
@@ -1,17 +1,49 @@
 package qna.domain;
 
+import java.sql.Timestamp;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import org.hibernate.annotations.CreationTimestamp;
+import org.springframework.data.annotation.CreatedDate;
 import qna.NotFoundException;
 import qna.UnAuthorizedException;
 
 import java.util.Objects;
 
-public class Answer {
+/**
+ * create table answer
+ * (
+ *     id          bigint generated by default as identity,
+ *     contents    clob,
+ *     created_at  timestamp not null,
+ *     deleted     boolean   not null,
+ *     question_id bigint,
+ *     updated_at  timestamp,
+ *     writer_id   bigint,
+ *     primary key (id)
+ * )
+ */
+@Entity
+public class Answer extends BaseTimeEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
-    private Long writerId;
-    private Long questionId;
+
+    @Column(columnDefinition = "clob")
```

Comment:
```suggestion
    @Lob
```

CLOB를 위한 애노테이션을 사용해볼 수 있겠네요 ㅎㅎ